--- origsrc/rake-0.8.7/install.rb	2008-11-02 12:58:29.000000000 -0600
+++ src/rake-0.8.7/install.rb	2010-09-04 22:01:21.503206700 -0500
@@ -41,11 +41,11 @@ def installBIN(from, opfile)
   File::unlink(tmp_file)
 end
 
-$sitedir = CONFIG["sitelibdir"]
+$sitedir = CONFIG["vendorlibdir"]
 unless $sitedir
   version = CONFIG["MAJOR"]+"."+CONFIG["MINOR"]
   $libdir = File.join(CONFIG["libdir"], "ruby", version)
-  $sitedir = $:.find {|x| x =~ /site_ruby/}
+  $sitedir = $:.find {|x| x =~ /vendor_ruby/}
   if !$sitedir
     $sitedir = File.join($libdir, "site_ruby")
   elsif $sitedir !~ Regexp.quote(version)
